source/compiler/qsc_frontend/src/incremental/tests.rs

```diff
@@ -58,6 +58,7 @@ fn one_callable() {
                         opens: {},
                         tys: {},
                         terms: {},
+                        importables: {},
                         vars: {},
                         ty_vars: {},
```

Comment:
We have tests here where the `importables` is empty. Are tests where this is set is covered in other kinds of tests from incremental?

Reply:
Oh wait I found one. https://github.com/microsoft/qsharp/blob/305225f8117d86b5989cea56fdc38e755fc10fb0/source/compiler/qsc/src/interpret/tests.rs#L2194-L2195

With that one case, I honestly think we're covered. Test validates that imports work in notebook cells. The other intricacies of import behavior is covered in the name resolution tests.